### ISSUE TITLE
Timepicker shows invalid time with UTC

### ISSUE THREAD

**Issue by bobrik** [grafana/grafana#5078]
Grafana built from tip:

```
Version: 3.1.0, Commit: v3.0.2+36-g07d3558, Build date: 2016-05-18 06:54:24 +0100 BST
```

My local timezone is UTC+1 (BST), I'm running Chrome on OSX.

1. Run Grafana.
2. Create new dashboard, set timezone to UTC.
3. Open dashboard with query string: `?from=1463531009968&to=1463536295329`
4. Observe invalid "From" and "To" time in timepicker (1 hour in the future), while graph shows correct time.

![screen shot 2016-05-18 at 10 30 09](https://cloud.githubusercontent.com/assets/89186/15354045/aa66246a-1ce3-11e6-9942-864781d7477c.png)

Supplied timestamps:

* From: `Wed May 18 00:23:29 UTC 2016`
* To: `Wed May 18 01:51:35 UTC 2016`

**Comment by bobrik** [comment]
Oh, the tooltip is correct:

<img width="565" alt="screen shot 2016-05-18 at 10 36 41" src="https://cloud.githubusercontent.com/assets/89186/15354213/75d8ced6-1ce4-11e6-8aa4-ec8d7dbaa854.png">

**Comment by torkelo** [comment]
thanks for  reporting this! fixed! Will be included in 3.0.3 patch release

**Comment by bobrik** [comment]
I was this close to fixing it myself 👍
